### DISCUSSION TITLE
能创建个交流群吗或者作者加我一个QQ

### DISCUSSION THREAD

**Discussion by OnZeng** [coder-hxl/x-crawl#28] [General]
作者能加一下我的QQ2528358258，有些问题想请教一下

**Comment by coder-hxl** [comment]
尽量在这里问吧，有些东西不方便在微信或QQ聊

**Comment by coder-hxl** [comment]
新增了 Discord 社区，可以通过 [Discord](https://discord.gg/SF7aaebg4E) 与其他 x-crawl 用户实时提问和讨论。
